docs/cmdlets-reference/restore-cohesitymssqlobject.md

```diff
@@ -252,6 +252,7 @@ Accept wildcard characters: False
 Specifies the secondary data filename pattern and corresponding directories of the DB.
 Secondary data files are optional and are user defined. The recommended file extension for secondary files is ".ndf".
 If this option is specified and the destination folders do not exist they will be automatically created.
+This field can be set only if restoring to a different target host.
```

Comment:
Can you put a todo here , we should support same host too unless that's api limitation

Reply:
This flag will not go with the same host.